quickstart/app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Product;
+use App\Models\Category;
+use App\Http\Requests\Product\StoreRequest;
+use App\Http\Requests\Product\UpdateRequest;
+
+class ProductController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $title = __('list') . ' ' . __('ps');
+        $products = Product::all();
+
+        return view('products.index', compact('title', 'products'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $title = __('addPs');
+        $categories = Category::all();
+
+        return view('products.create', compact('title', 'categories'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(StoreRequest $request)
+    {
+        $product = Product::whereRaw('product_name = ? and category_id = ?',
+            [$request->product_name, $request->category_id,])->first();
```

Comment:
```suggestion
        $product = Product::whereRaw('product_name = ? and category_id = ?', [
            $request->product_name,
            $request->category_id,
        ])->first();
```
sửa lại convention mảng nhé